### DISCUSSION TITLE
Can Nightingale with Categraf write data into InfluxDB?

### DISCUSSION THREAD

**Discussion by HAHA-LIU** [ccfos/nightingale#1329] [General]
1. 使用influxdb，webapi和server配置文件如何修改。

**Comment by UlricQin** [comment]
现在只能写influxdb，不能读。之所以能写，是因为influxdb支持remote write接口，把influxdb的remote write接口配置到n9e-server中，n9e-server 就可以把数据写入influxdb。但是influxdb不支持Prometheus querier接口，所以没法作为数据源来读。后面v6版本会考虑直接支持influxdb的读
